parameters.json

```diff
@@ -0,0 +1,122 @@
+{
```

Comment:
why did you add these into this code base?

Reply:
@dignifiedquire 

> why did you add these into this code base?

CircleCI uses this file (see the .circleci/config.yaml in this PR) to download Groth parameters. Previously, we were `cargo install`'ing paramcache and generating them ourselves based on a rust-fil-proofs Git SHA, which was a nightmare.